services/ai-analytics/log-ingestion.mdx

```diff
@@ -0,0 +1,256 @@
+---
+title: "AI Analytics Log Ingestion"
+description: "Send web server access logs to Edgee for AI bot traffic analysis. Complete setup guide for log ingestion across different hosting providers."
+---
+
+import EdgeeSdk from '/snippets/edgee-sdk.mdx';
+
+<EdgeeSdk />
+
+# AI Analytics Log Ingestion
+
+Edgee's AI Analytics helps you understand AI bot traffic on your website through our log ingestion service. Send your web server access logs to our endpoint and gain insights into how AI bots and LLM crawlers interact with your content.
+
+## Prerequisites
+
+Before setting up log ingestion, you'll need:
+- An active Edgee account
+- A project configured for AI Analytics
+- Access to your web server's access logs or logging system
+
+## Quick Start
+
+### 1. Get Your Project Configuration
+
+First, retrieve your project ID from the Edgee console:
+1. Navigate to your project dashboard
+2. Go to the AI Analytics section
+3. Copy your unique Project ID
+
+### 2. Configure Your Logging Endpoint
+
+Send logs to the following endpoint:
+
+```
+POST https://logs.edgee.app/{PROJECT_ID}
+```
+
+Replace `{PROJECT_ID}` with your actual project ID.
+
+### 3. Set Content Type
+
+All log submissions must use:
+```
+Content-Type: text/plain
+```
+
+## Log Format Specification
+
+### Required Fields
+
+Each log entry must be a JSON object containing these fields:
+
+| Field | Type | Description | Example |
+|-------|------|-------------|---------|
+| `timestamp` | string | ISO 8601 format (YYYY-MM-DDTHH:MM:SSZ) | `"2024-01-01T00:00:00Z"` |
+| `ip` | string | Client IP address | `"192.168.1.1"` |
+| `ua` | string | User-Agent header | `"Mozilla/5.0 (Windows NT 10.0; Win64; x64)"` |
+| `method` | string | HTTP method | `"POST"`, `"GET"`, `"PUT"` |
+| `path` | string | Request path | `"/api/chat"`, `"/api/completion"` |
+| `status` | string | HTTP status code | `"200"`, `"404"`, `"500"` |
+| `content_type` | string | Response content type | `"application/json"` |
+| `host` | string | Host header | `"api.mycompany.com"` |
+
+### Optional Fields
```

Comment:
we should explain that even if this field is optional, it highly recommended to be able to measure the traffic provenance

Reply:
👍